README.md

```diff
@@ -1,39 +1,270 @@
+
+
+
+____________________________________________________
+____________________________________________________
+____________________________________________________
+____________________________________________________
+Accuracy Score for DecisionTreeClassifier is 
+0.9622641509433962
+
+
+Confusion Matrix for DecisionTreeClassifier is 
+[[16  0  0]
+ [ 0 20  1]
+ [ 0  1 15]]
 
-Please make sure to update tests as appropriate.
 
-## License
-[MIT](https://choosealicense.com/licenses/mit/)
+Classification Report for DecisionTreeClassifier is 
+              precision    recall  f1-score   support
+
+           0       1.00      1.00      1.00        16
+           1       0.95      0.95      0.95        21
+           2       0.94      0.94      0.94        16
+
+    accuracy                           0.96        53
+   macro avg       0.96      0.96      0.96        53
+weighted avg       0.96      0.96      0.96        53
+
+
+
+____________________________________________________
+____________________________________________________
+____________________________________________________
+____________________________________________________
+Accuracy Score for KNeighborsClassifier is 
+0.9811320754716981
+
+
+Confusion Matrix for KNeighborsClassifier is 
+[[16  0  0]
+ [ 0 20  1]
+ [ 0  0 16]]
+
+
+Classification Report for KNeighborsClassifier is 
+              precision    recall  f1-score   support
+
+           0       1.00      1.00      1.00        16
+           1       1.00      0.95      0.98        21
+           2       0.94      1.00      0.97        16
+
+    accuracy                           0.98        53
+   macro avg       0.98      0.98      0.98        53
+weighted avg       0.98      0.98      0.98        53
+
+
+
+____________________________________________________
+____________________________________________________
+____________________________________________________
+____________________________________________________
+Accuracy Score for LogisticRegression is 
+0.9811320754716981
+
+
+Confusion Matrix for LogisticRegression is 
+[[16  0  0]
+ [ 0 20  1]
+ [ 0  0 16]]
+
+
+Classification Report for LogisticRegression is 
+              precision    recall  f1-score   support
+
+           0       1.00      1.00      1.00        16
+           1       1.00      0.95      0.98        21
+           2       0.94      1.00      0.97        16
+
+    accuracy                           0.98        53
+   macro avg       0.98      0.98      0.98        53
+weighted avg       0.98      0.98      0.98        53
+
+
+
+____________________________________________________
+____________________________________________________
+Included special classifier with fixed nature
+2020-05-28 15:25:28.257871: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
```

Comment:
i think you should take this out. the warnings

Reply:
Done.